LCM/Base_Inventory.mof

```diff
@@ -23,6 +23,21 @@ instance of MSFT_nxServiceResource
 
 };
 
+instance of MSFT_nxFileInventoryResource
```

Comment:
Is Base_Inventory still used? or the change_tracking_inventory.mof is used directly?